### PR TITLE
cmd/migrate-db: Update agent migration

### PR DESCRIPTION
**Pull request by mhilton**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candid/53/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candid/54/

File: cmd/migrate-db/internal/migrate.go [hunk 1]
```diff
@@ -58,10 +58,22 @@ func Copy(ctx context.Context, dst store.Store, src Source) error {
...
 		// The ID field is store specific, so cannot be copied between them.
 		identity.ID = ""
-		err := dst.UpdateIdentity(ctx, identity, update)
-		if err != nil {
-			log.Printf("cannot update user %s: %s", identity.Username, err)
-			failed = true
+		destIdentity := store.Identity{
+			ProviderID: identity.ProviderID,
+		}
+		if err := dst.Identity(ctx, &destIdentity); err != nil {
```
Comment (rogpeppe) [inline cmd/migrate-db/internal/migrate.go]: A comment stating why we're doing this might be good, please.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candid/55/

**Comment by mhilton** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.theblues.io:8080/job/candid-merge

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candid/57/

**Pull request merged**